interface1.html

```diff
@@ -900,1291 +903,1293 @@ <h3 style="margin-top: 15px; margin-bottom: -5px">Specify terminals
+				<div class="constraint-row">
+					<span><input type="checkbox" name="constraints" value="group">Group</span>
+					<div class="info">
+						<span class="content">Assign a violation for every c-pair that is not contained in at least one &phi;. SPOT function: group(). (Kalivoda 2018)</span>
+					</div>
+				</div>
+				<div class="category-row">
+					<div style="white-space:nowrap;" class="option-selection-div custom-option-div"><input type="checkbox" name="option-group" value="requireMinimal">Require same parent &phi;
+						<div class="info">
+							<span class="content">Assign a violation for every c-pair<br/>that is not directly dominated by<br/>the same parent &phi;. (Kalivoda 2018)</span>
 						</div>
 					</div>
+				</div>
+				<div class="category-row">
+					<div style="white-space:nowrap;" class="option-selection-div custom-option-div"><input type="checkbox" name="option-group" value="requireAdjacent">Enforce only for linearly adjacent words</div>
+				</div>
+			</div>
 
-					<div class="constraint-selection-table">
-						<input class="checkbox" checked="true" name="category-antiCCPhi" value="phi" style="display:none;"/>
-						<div class="constraint-row">
-							<span><input type="checkbox" name="constraints" value="antiCCPhi">Anti-C-Command-to-&phi;</span>
-							<div class="info">
-								<span class="content">If x<sup>0</sup> and y<sup>0</sup> are mutually non-commanding, then assign a violation if there is no &phi; containing ω<sub>x</sub> and excluding ω<sub>y</sub>, and a violation if there is no &phi; containing ω<sub>y</sub> and excluding ω<sub>x</sub>. SPOT function: antiCCPhi(). (Kalivoda 2018)</span>
-							</div>
+		</fieldset>
+
+		<br/>
+
+		<fieldset>
+
+			<legend><h2>Ordering <span class="arrow"></h2></legend>
+
+			<div class="constraint-selection-table">
+				<div class="constraint-row">
+						<span><input type="checkbox" name="constraints" value="noShift">NoShift</span>
+						<div class="info"><span class="content">Assign a violation if the terminals in the prosodic tree do not maintain the same precedence relations as those in the syntactic tree. (Bennett et al. 2016)</span></div>
+				</div>
+			</div>
+
+			<div class="constraint-selection-table">
+				<div class="constraint-row">
+						<span><input type="checkbox" name="constraints" value="noShiftGradient">Linearity</span>
+						<div class="info"><span class="content">For every pair of terminals (x,y), assign a violation if x precedes y in the terminal string of the syntactic tree and y precedes x in the terminal string of the prosodic tree. (McCarthy & Prince 1995)</span></div>
+				</div>
+			</div>
+
+		</fieldset>
+
+		<br/>
+			</div>
+		<div class="spotBlock">
+			<h2>Markedness constraints<div class="info"><span class="content">Markedness constraints refer only to outputs, not to inputs (Prince & Smolensky 1993/2004). In SPOT, this means that they inspect only the pTree in each candidate &#12296;sTree, pTree&#12297;</span></div></h2>
```

Comment:
Not vital: I think this would be clearer to the new SPOT user if "pTree" was replaced with "prosodic tree" or "output"